### PR TITLE
Auth Pages UI. Closes #23

### LINKED ISSUE #23: Auth Pages UI
- Refer this figma design https://www.figma.com/file/OtcEA7dW5XD31ex5sO5aDB/Mobile?node-id=8%3A101

### PR DESCRIPTION
**Pull request by Sudhindra3000**
### Todo:

- Some form validation stuff
- Prefix icon in Referral code field in Register page
- Fill colors are not showing properly in Pincode text field
- ~Border color in OutlinedButtons~

Minor stuff. Can be done later

### CODE REVIEW

File: lib/pages/landing_page.dart [hunk 1]
```diff
@@ -86,5 +100,7 @@ class LandingPage extends StatelessWidget {
   void onLoginClick(BuildContext context) =>
       Navigator.of(context).pushNamed('/login');
 
+  void onHeartClick() => launch(helikopterHelikopter);
```
Comment (dhruv1294) [inline lib/pages/landing_page.dart:103]: lol

File: lib/constants/urls.dart [hunk 2]
```diff
@@ -0,0 +1,3 @@
+const deltaUrl = 'https://delta.nitt.edu/';
+
```
Comment (dhruv1294) [inline lib/constants/urls.dart:2]: do we need seperate files for all string constants??

**Review by dhruv1294 — APPROVED** [review]

File: lib/theme/buttons.dart [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  shape: buttonShape,
+);
+
+// TODO: border color is not showing 😢
+final outlinedButtonStyle = OutlinedButton.styleFrom(
+  textStyle: buttonTextStyle,
+  minimumSize: buttonMinSize,
+  elevation: 0,
+  shape: outlinedButtonShape,
+);
```
Comment (dhruv1294) [inline lib/theme/buttons.dart:47]: https://stackoverflow.com/questions/57874576/how-to-change-outlinedbutton-border-color 
maybe use this??

Comment (Sudhindra3000) [reply lib/theme/buttons.dart:47]: Thanks it's working now. I'll update it

File: lib/constants/urls.dart (hunk 2 above)
Comment (Sudhindra3000) [reply lib/constants/urls.dart:2]: We might add a lot of string constants in future. So it's better to separate them

**Review by satyamurti — APPROVED** [review]

**Pull request merged**